README.md

```diff
@@ -91,7 +92,7 @@ module "spectral_lambda_integration" {
 
   # Environment variables used by the integration
   env_vars = {
-    # Mandatory - Your spectral DSN retrieved from SpectralOps
+    # Mandatory (unless you are using vault) - Your spectral DSN retrieved from SpectralOps
```

Comment:
the spectral_dsn is not mandatory when using secret menages? this is the intention

Reply:
Not mandatory as env var, this is under the env vars variable